src/org/sosy_lab/java_smt/solvers/yices2/Yices2NativeApiTest.java

```diff
@@ -201,6 +204,8 @@ public void rationalError() {
     System.out.println(rat); // "use" variable
   }
 
+  // TODO: what is this test supposed to be doing? And remove print.
+  @Ignore
```

Comment:
Most of those tests are simply tests for loading and running Yices. They serves as examples and were helpful for debugging initial issues. If the tests do no longer bring benefit, please remove them, and do not just disable them.

Reply:
I reworked all tests with printouts into assertions and re-enabled all. I also found some issues and fixed them, added documentation and 2 more native tests.